packages/core/core/create-handler.js

```diff
@@ -20,6 +21,15 @@ const createHandler = (optionByName = {}) => {
         try {
             initDebugLog(eventName, event);
 
+            const requestMethod = event.httpMethod;
+            const requestPath = event.path;
+            if (requestMethod && requestPath) {
```

Comment:
what's the use of this variables besides the loging?

Reply:
Just logging

(Thought it was useful context in another project... could remove or really deserves a full revamp of logging/telemetry across Frigg)